daml-lf/transaction/src/main/scala/com/daml/lf/TransactionNodesStatistics.scala

```diff
@@ -54,11 +54,17 @@ final case class TransactionNodesStatistics(
   def nodes: Int = actions + rollbacks
 }
 
+final case class TransactionNodesStatistics(
```

Comment:
Is the intention that `com.daml.ledger.participant.state.v2.CompletionInfo` will have to emit two sets of `TransactionNodesStats`? @simonmaxen-da has pointed out some language in the spec that this is understood, but I am not sure if all readers of the spec understand that we are now "doubling" the number of metrics emitted - particularly glancing at this section this might surprise readers: https://docs.google.com/document/d/1HQyVLCANqw_l_gbxBzgOqsiwmIDNlwCRQffVYJnYTeU/edit#bookmark=id.gajjyxn7ia49

I.e. instead of having 9 integers added to every submitter-side-`TransactionAccepted`, we would be added 18 integers.

Reply:
Yes.  In the case of Canton this data does not _need_ to be persisted as the transaction observed by the submitter is identical to the original (and so statistics could be computed at Update construction time).

Reply:
Canton persists data emitted via the `ReadService` in case of crash recovery and in case the indexer ever asks to subscribe from an earlier offset.